src/Features/CSharp/Portable/ChangeSignature/CSharpChangeSignatureService.cs

```diff
@@ -356,41 +403,43 @@ private SyntaxNode GetNodeContainingTargetNode(SyntaxNode matchingNode)
                     isReducedExtensionMethod = true;
                 }
 
-                var newArguments = PermuteArgumentList(document, declarationSymbol, invocation.ArgumentList.Arguments, signaturePermutation, isReducedExtensionMethod);
+                SignatureChange signaturePermutationWithoutAddedParameters = signaturePermutation.WithoutAddedParameters();
```

Comment:
I think that this is the only case affected by `this `and `params `parameters like skipping `this `and supporting multiple `params`. .It would be great to refactor this but now for now.